src/OpenTelemetry.Exporter.Jaeger/JaegerExporterOptions.cs

```diff
@@ -80,9 +81,10 @@ public JaegerExporterOptions()
         public int AgentPort { get; set; } = 6831;
 
         /// <summary>
-        /// Gets or sets the Jaeger HTTP endpoint. Default value: "http://localhost:14268".
+        /// Gets or sets the Jaeger HTTP endpoint. Default value: "http://localhost:14268/api/traces".
+        /// Typically https://jaeger-server-name:14268/api/traces.
         /// </summary>
-        public Uri Endpoint { get; set; } = new Uri("http://localhost:14268");
+        public Uri Endpoint { get; set; } = new Uri(DefaultJaegerEndpoint);
```

Comment:
I think this is a breaking behavior. Previously /api/traces was appended to the endpoint user provided. Now we expect the user to provide the whole url.

(should be fine as Endpoint was added only in 1.2 and we haven;t yet released a stable) 

@CodeBlanch Could you comment?

Also, we should update the readme to reflect the new change.

Reply:
Agreed that this a breaking change. As I said in my PR comment, I personally think it makes sense to have this work like the zipkin exporter does - but I totally understand if you would like to avoid a breaking change. This is fairly trivial to change to some other pattern (like maybe another property on the `JaegerExporterOptions` with the path to append to the endpoint, and have it default to `/api/traces`).
I'll update the readme as a part of this PR to reflect the decision you make on how to implement this.

Reply:
Hi @cijothomas just wanted to check if you've had a chance to discuss with the other maintainers what you'd like me to do here? Thanks!

Reply:
It may be good to confront it with the specification and how other languages are doing it (I suspect inconsistency).

Reply:
So in the [Environment variable docs](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#jaeger-exporter) it just lists `http://localhost:14250` as the default value, and the [`jaeger` docs](https://www.jaegertracing.io/docs/1.31/apis/#thrift-over-http-stable) are explicit about the path to use for Thrift over HTTP: `/api/traces`. I guess where this breaks down for my organization is that we don't use the jaeger agent, but rather a third party one that fails when we use this combination of values.

I've done some investigation on other otel languages:
* Java - it looks like it only supports grpc
* Javascript - Namely is already successfully sending metrics to our agent using (It looks like otel js is using the `jaeger-client` npm package, which just uses the `endpoint` as-is):
```js
const exporter = new JaegerExporter({
  endpoint: 'http://our-trace-collector:9080/v1/trace',
});
```
* Golang - according to their [readme](https://github.com/open-telemetry/opentelemetry-go/tree/v1.3.0/exporters/jaeger#configuration) and [docs](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/jaeger#WithEndpoint), the endpoint can be specified with a path (`OTEL_EXPORTER_JAEGER_ENDPOINT` defaults to `http://localhost:14268/api/traces`), and looking through the code, it just uses the `endpoint` value specified
* Ruby - [readme](https://github.com/open-telemetry/opentelemetry-go/blob/bd817df68c274fbace901fab63da2bea267315ab/exporters/jaeger/uploader.go#L294) and [code](https://github.com/open-telemetry/opentelemetry-ruby/blob/980f0e454c59e822de439635ee513e2bcdf820b8/exporter/jaeger/lib/opentelemetry/exporter/jaeger/collector_exporter.rb#L28) indicate that they export to the URL specified in the env var as-is

I haven't looked through any of the other languages, but it does seem that my change is more consistent with at least a few of the other languages that otel supports (and incidentally all the ones my Company uses 😄).

Reply:
@abe545 I think that .NET SDK does it correctly. You may try discussing it with the [OTel spec SIG](https://github.com/open-telemetry/community#special-interest-groups) if it bothers you. Especially that Jaeger exporter is a quite popular one and having inconsistencies between languages can be frustrating.

Reply:
FYI https://github.com/open-telemetry/opentelemetry-go/issues/2599

Reply:
One other thing I forgot to mention is that the behavior in this repo is different from the [OpenTracing jaeger client](https://github.com/jaegertracing/jaeger-client-csharp) (which we're migrating from). It also just used the endpoint property/env-var as-is.
If you want to keep the behavior as-is, would you have a problem with me adding a new configuration option to set the path? Right now there is no way for me to use jaeger with open telemetry in my organization for .net. That is the problem I'm trying to solve, and I'm not terribly concerned about *how* to solve it.

Reply:
Now that it looks like the spec will be updated to make it clear that the endpoint should include the path, how do you feel about this pr @pellared and @cijothomas (also, do I need to do something to kick off the build)?

Reply:
@abe545 In general I prefer the approach of your PR (this is also how I implemented it in OTel Go SDK some time ago 😄). The change is not really breaking from the perspective as there was no stable release.